servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -140,6 +140,10 @@
   @SuppressFBWarnings("URF_UNREAD_FIELD")
   private Future<?> mLostFilesDetectionService;
 
+  /**
+   * This records innodesfiles with ttl set in the corresponding ttlbucket,for the
```

Comment:
rename "records" -> "maintains" and "innodesfiles" -> "inodes"

also, add a space after the comma ","